src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -797,8 +797,8 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
                 logger.debug(`Member(s) have left: queueing sender key rotation`);
                 this.makeNewKeyTimeout = setTimeout(this.onRotateKeyTimeout, MAKE_KEY_DELAY);
             } else if (anyJoined) {
-                logger.debug(`New member(s) have joined: queueing sender key rotation`);
-                this.makeNewKeyTimeout = setTimeout(this.onRotateKeyTimeout, MAKE_KEY_DELAY);
+                logger.debug(`New member(s) have joined: re-sending keys`);
+                this.requestKeyEventSend();
```

Comment:
I am wondering if we want to rename this to `requestCurrentKeySend()`.
With the logger content it is understandable but this would make it more clear in other places.

Reply:
I've changed to `requestSendCurrentKey()`.